EIPS/eip-3540.md

```diff
@@ -1,6 +1,6 @@
 ---
 eip: 3540
-title: EVM Object Format (EOF) v1
+title: EOF - EVM Object Format (EOF) v1
```

Comment:
This seems duplicates, is this what you intends?

```suggestion
title: EOF - EVM Object Format v1
```

Reply:
It is slightly redundant in this case, but the goal is to have consistent prefixing for ease of searching.

In this case ...

EOF: EVM Object Format (EOF) v1

Is fine, imho, as the two EOFs in that title serve different purposes.

Anyway - better as a new PR from @chfast, I think, to redo them all.